internal/collector/instance.go

```diff
@@ -108,5 +140,26 @@ func AddToPod(
 	}
 
 	outPod.Containers = append(outPod.Containers, container)
-	outPod.Volumes = append(outPod.Volumes, configVolume)
+}
+
+// startCommand generates the command script used by the collector container
+func startCommand(includeLogrotate bool) []string {
+	var startScript = `
+/otelcol-contrib --config /etc/otel-collector/config.yaml
+`
+
+	if includeLogrotate {
```

Comment:
We need a way to make sure that the logrotate.conf is valid and present before calling logrotate -- I mean, currently, above we only mount the config is it passes

```
includeLogrotate && spec != nil && spec.Logs != nil && spec.Logs.RetentionPeriod != nil
```

But we alter the startCommand just based on the `includeLogrotate` boolean.

Reply:
Yeah I need to remove the checking for `spec` and whatnot on line 110.

Reply:
Not sure how we would check that `logrotate.conf` is present and valid before calling `logrotate` :thinking: